install/concurrency/docs/CONCURRENCY2.0-ReleaseNotes.html

```diff
@@ -77,22 +77,21 @@ <h2><a name="platform">Platform Notes</a></h2>
 <p>The Jakarta Concurrency TCK tests have been run on the following platforms:</p>
 <ul>
 <li>MAC OS X, <em>El Capitan</em> (10.9.5+)</li>
```

Comment:
I would remove ```MAC OS X``` unless we have heard from someone that was actually used to pass the Jakarta Concurrency 2.0 TCK tests with.

Reply:
I pulled it out. If lead wishes, we can change it back